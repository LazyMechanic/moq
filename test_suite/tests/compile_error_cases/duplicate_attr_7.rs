const fn def() -> i32 {
    2
}

#[moq::automock]
trait Trait {
    #[moq(default_with = "def")]
    #[moq(default = 1)]
    const CONST: i32;
}

fn main() {}
