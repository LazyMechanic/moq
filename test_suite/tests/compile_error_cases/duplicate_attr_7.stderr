error: conflict moq attributes: `default_with` and `default`
 --> tests/compile_error_cases/duplicate_attr_7.rs:8:11
  |
8 |     #[moq(default = 1)]
  |           ^^^^^^^^^^^
